dev.overlayed.Overlayed.yaml

```diff
@@ -0,0 +1,96 @@
+# useful info for tauri v1 https://github.com/tauri-apps/tauri/discussions/4426
+
+id: dev.overlayed.Overlayed
+
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+
+rename-icon: Overlayed
+command: overlayed
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=x11
```

Comment:
You don't need all 3. The previous comment still applies

Reply:
Was a mistake, I've fixed it to use `--socket=wayland` and `--socket=fallback-x11`.